python/ql/src/experimental/semmle/python/frameworks/Yaml.qll

```diff
@@ -0,0 +1,49 @@
+/**
+ * Provides classes modeling security-relevant aspects of the `yaml` package.
+ */
```

Comment:
PyYAML is a bit different than most other PyPI packages, since it provides a (python) package with a different name. I think we should at least note that the PyPI package is called PyYAML here.

Reply:
Done